api/types/container/container_top.go

```diff
@@ -0,0 +1,21 @@
+package container
+
+// ----------------------------------------------------------------------------
+// DO NOT EDIT THIS FILE
+// This file was generated by `swagger generate operation`
+//
+// See hack/swagger-gen.sh
+// ----------------------------------------------------------------------------
+
+// ContainerTopOKBody container top o k body
```

Comment:
yes for the comment, it can come later.
I also meant: the method name seem strange to me. but it's fine.